src/app/_components/_sharedcomponents/Preferences/PreferencesSubElementVariants/AnimationOptionsTab.tsx

```diff
@@ -0,0 +1,343 @@
+            fastAnimations
+        };
+
+        try {
+            const result = await savePreferencesGeneric(
+                user,
+                { animations: animationPreferences },
+                updateUserPreferences
+            );
+
+            if (result.success) {
+                setOriginalMode(animationMode);
+                setOriginalFastAnimations(fastAnimations);
+                setSaveStatus(SaveStatus.Success);
+                setSaveMessage('Animation preferences saved successfully.');
+                setTimeout(() => setSaveStatus(SaveStatus.NoChange), 3000);
+                setHasChanges(false);
+                if (setHasNewChanges) {
+                    setHasNewChanges(false);
+                }
+            } else {
+                setSaveStatus(SaveStatus.Error);
+                setSaveMessage('Failed to save preferences to server.');
+            }
+        } catch (error) {
+            console.error('Failed to save animation preferences:', error);
+            if (error instanceof Error) {
+                setSaveMessage(error.message);
+            } else {
+                setSaveMessage('Unknown Server Error');
+            }
+            setSaveStatus(SaveStatus.Error);
+        } finally {
+            setIsSaving(false);
+        }
+    };
+
+    return (
+        <>
+            <Box sx={styles.functionContainer}>
+                <Typography sx={styles.typographyContainer} variant={'h2'}>Animation Settings</Typography>
+                <Divider sx={{ mb: '20px' }}/>
+
+                <FormControl component="fieldset">
+                    <RadioGroup
+                        value={animationMode}
+                        onChange={handleModeChange}
+                        sx={styles.radioGroup}
+                    >
+                        <FormControlLabel
+                            value="all"
+                            control={<Radio />}
+                            label={
+                                <Box>
+                                    <Typography variant="body1" fontWeight="bold">Show All Animations</Typography>
+                                    <Typography variant="body2" color="text.secondary">
+                                        Full visual experience with all animations enabled.
+                                    </Typography>
+                                </Box>
+                            }
+                            sx={styles.radioOption}
+                        />
+                        <FormControlLabel
+                            value="onlyDamage"
+                            control={<Radio />}
+                            label={
+                                <Box>
+                                    <Typography variant="body1" fontWeight="bold">Show Only Damage Animations</Typography>
+                                    <Typography variant="body2" color="text.secondary">
```

Comment:
These subtitle Typography components need to have a different color, they're currently defaulting to black and indistinguishable from the background

Reply:
color: '#878787' is what the other tabs are using. The theme could be updated to use that color as `text.secondary` too.